ckan/saml2/who.ini

```diff
@@ -0,0 +1,38 @@
+[plugin:auth_tkt]
+use = repoze.who.plugins.auth_tkt:make_plugin
+# For development only. DO NOT USE in any other environment.
+secret = development-only-secret
+
+[plugin:saml2auth]
+use = saml2.s2repoze.plugins.sp:make_plugin
+saml_conf = ckanext.saml2.config_login_gov.sp_config
+remember_name = auth_tkt
+identity_cache = memcached
+sid_store = outstanding
+sid_store_cert = sid_store_cert
```

Comment:
You don't need this if you are going to use the saml2auth extension, since we don't use s2repoze in the new plugin. Also the s2repoze plugin of pysaml2 is not even maintained anymore, so it's not recommended.